### PR TITLE
Fix network interface spec collector on Windows

### PR DESCRIPTION
**Pull request by itchyny**
This pull request fixes network interface spec on Windows.

### CODE REVIEW

File: spec/windows/interface.go [hunk 1]
```diff
@@ -59,6 +59,8 @@ func (g *InterfaceGenerator) Generate() ([]spec.NetInterface, error) {
 			switch v := addr.(type) {
 			case *net.IPAddr:
 				ip = v.IP
+			case *net.IPNet:
```
Comment (astj) [inline spec/windows/interface.go]: Should we consider range of v.Mask?

**Comment by astj** [comment]
I don't know what subnet of an interface mean, and whether v.IP is proper representation of the interface's IP address...

**Comment by itchyny** [comment]
I think subnet mask is commonly used in local network. The network interface spec collector for Linux also ignores subnet masks ([ref](https://github.com/mackerelio/mackerel-agent/blob/fbbe8d3e086fddaabf5c163165e977251551ea70/spec/linux/interface.go#L60)). I'm not sure users want to see subnet masks or network addresses, but it's reasonable to have the same implementation on each architectures.

File: spec/windows/interface.go (hunk 1 above)
Comment (mattn) [reply spec/windows/interface.go]: How about replace this to `net.Parse(addr.String())` ?

**Comment by astj** [comment]
Okay, I understand.

File: spec/windows/interface.go (hunk 1 above)
Comment (itchyny) [reply spec/windows/interface.go]: `net.ParseIP` does not accept addresses with subnet mask.

**Comment by itchyny** [comment]
Thank you.

**Pull request merged**